### DISCUSSION TITLE
Problema di modifica pubblicazioni su Zigbee2mqtt

### DISCUSSION THREAD

**Discussion by lnrd00** [indomus/forum#2889] [ZigBee, Z-Wave, Thread, Matter]
Salve, avrei una domanda. Ho un sensore di temperatura e umidità Sonoff ZigBee SNZB-02, già connesso da un bel po' a Home Assistant, e ho già configurato Zigbee2mqtt seguendo la guida inDomus. Sto cercando di modificare il tempo con cui pubblica il valore di umidità, purtroppo quando ci provo ottengo il seguente errore:
<img width="1280" alt="Schermata 2024-02-06 alle 15 04 17" src="https://github.com/indomus/forum/assets/154708323/e4dc32bf-fe04-408d-8bf2-9d5d66e73026">
Cosa potrebbe essere? Grazie in anticipo!
P.S. L'errore si presenta anche se provo ad effettuare una variazione ad un parametro differente da quello di umidità, come ad esempio, la temperatura.

Per completezza, vi incollo anche i logs:

```
Zigbee2MQTT:error 2024-02-06 15:14:23: Request 'zigbee2mqtt/bridge/request/device/configure_reporting' failed with error: 'Bind 0x00124b002a534570/1 msRelativeHumidity from '0xe0798dfffeb8dd9a/1' failed ({"address":48924,"clusterId":32801,"sequence":9} after 10000ms)'
Zigbee2MQTT:info  2024-02-06 15:14:23: MQTT publish: topic 'zigbee2mqtt/bridge/response/device/configure_reporting', payload '{"data":{},"error":"Bind 0x00124b002a534570/1 msRelativeHumidity from '0xe0798dfffeb8dd9a/1' failed ({\"address\":48924,\"clusterId\":32801,\"sequence\":9} after 10000ms)","status":"error","transaction":"33xuz-1"}'
```

**Comment by snakuzzo** [comment]
Devi cercare sul progetto ufficiale e aprire eventualmente una issue

**Reply by KC-inDomus** [reply]
(cmq a 5 secondi di polling non ce lo puoi mettere, minimo 10 che io sappia)

**Reply by Scialla** [reply]
cmq, su un device a batteria che si risveglia dal deepsleep al variare del parametro (temperatura o umidità) comunicandolo, cambiare il tempo di polling non ha senso (IMHO). 
Questa è una cosa che si può fare in device alimentati non in quelli a batteria. Se anche riuscissi a farlo, crollerebbe drasticamente la durata della batteria e non avrebbe senso continuare a comunicare ogni tot secondi lo stesso valore

**Comment by NUTI76** [comment]
devi risvegliare il dispositivo premendo il pulsante laterale.
